frontend/src/sections/JoinCommunity/JoinCommunity.tsx

```diff
@@ -27,115 +27,135 @@ function Join() {
           </p>
         </div>
 
-        <div className='w-96 md:w-auto mt-10 relative'>
```

Comment:
@devendra0009 I recommend you explore mobile first approach for responsive design for restructuring this section.